nifi-nar-bundles/nifi-extension-utils/nifi-processor-utils/src/main/java/org/apache/nifi/processor/util/bin/BinFiles.java

```diff
@@ -94,6 +96,16 @@
             .addValidator(StandardValidators.createTimePeriodValidator(1, TimeUnit.SECONDS, Integer.MAX_VALUE, TimeUnit.SECONDS))
             .build();
 
+    public static final PropertyDescriptor ADD_MERGED_UUID = new PropertyDescriptor.Builder()
```

Comment:
Agree, this is just adding configuration complexity for very little (none) benefit. I just pushed a commit to remove this parameter.